docker/Dockerfile-workers

```diff
@@ -26,6 +26,11 @@ COPY ./docker/prefix-log /usr/local/bin/
 # Expose nginx listener port
 EXPOSE 8080/tcp
 
+# Default to starting Redis.
+# (This environment variable is only used in the Complement image to disable
+# Redis when workers aren't being used)
+ENV START_REDIS=true
```

Comment:
why does this get an `ENV` line, but other things like `SYNAPSE_WORKER_TYPES` do not?

Reply:
So that this gets set by default (and the others don't).
In other words, to avoid breaking people that depend on this whilst still having an off switch.

Reply:
right, but should we just have `START_REDIS=""` mean "redis should be started" ?

(or possibly more intuitively, have a `NO_START_REDIS` env var which can be set to anything non-empty to inhibit redis)

Reply:
Rats, that was me forgetting to actually remove it, well spotted